portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,17 +26,24 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments = new ArrayList<String>();    
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String jsonComments = convertToJsonUsingGson(comments);
     ArrayList<String> messages = new ArrayList<String>();
     messages.add("Good morning.");
     messages.add("Good afternoon.");
     messages.add("Good evening.");
 
     String jsonMessages = convertToJsonUsingGson(messages);
+  }
 
-    response.setContentType("application/json;");
-    response.getWriter().println(jsonMessages);
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = request.getParameter("text-input");
+    comments.add(comment);
+    response.sendRedirect("/index.html");
```

Comment:
Why is this line necessary? Is it just refreshing index.html so that the new comment shows up on the page?

Reply:
Yes! That was my own preference. Because the comments only appear when the page loads, I wanted the user to see their own comment right away.